tests_ext/src/test/java/com/dirgub/klee0kai/stone/text_ext/cache/ExtEarthSwitchCacheTests.java

```diff
@@ -0,0 +1,80 @@
+package com.dirgub.klee0kai.stone.text_ext.cache;
+
+import com.github.klee0kai.stone.Stone;
+import com.github.klee0kai.test.di.swcache.SwitchCacheComponent;
+import com.github.klee0kai.test.mowgli.earth.Mountain;
+import com.github.klee0kai.test.mowgli.earth.River;
+import com.github.klee0kai.test_ext.inject.di.swcache.SwitchCacheExtComponent;
+import org.junit.jupiter.api.Test;
+
+import java.lang.ref.WeakReference;
+
+import static java.lang.Thread.sleep;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class ExtEarthSwitchCacheTests {
+
+
+    @Test
```

Comment:
write test 
 - created in DIPro, switch ref from DI
 - Create in DI, switch ref from DIPro

Reply:
done